tests/foreman/cli/test_satellitesync.py

```diff
@@ -16,95 +16,113 @@
-    assert result.return_code == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:mnt_t:s0' in result.stdout[0]
-
-    # Fix SELinux policy for new directory
-    result = ssh.command(f'semanage fcontext -a -t foreman_var_run_t "/mnt/{export_dir}(/.*)?"')
-    assert result.return_code == 0
-
-    result = ssh.command(f'restorecon -Rv /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    # Assert that we have the correct policy
-    result = ssh.command(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.return_code == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:foreman_var_run_t:s0' in result.stdout[0]
-
-    # Update the 'pulp_export_destination' settings to new directory
-    Settings.set({'name': 'pulp_export_destination', 'value': f'/mnt/{export_dir}'})
-    # Create an organization to reuse in tests
-    yield export_dir
-    ssh.command(f'rm -rf /mnt/{export_dir}')
-
-
-class ExportDirectoryNotSet(Exception):
-    """Raise when export Directory is not set or found"""
+    """Grab export directory at the end remove the export directory with
+    all exported repository archives"""
 
 
 @pytest.mark.run_in_one_thread
 class TestRepositoryExport:
     """Tests for exporting a repository via CLI"""
 
     @pytest.mark.tier3
-    def test_positive_export_custom_product(self, class_export_directory, module_org):
-        """Export a repository from the custom product
+    def test_positive_export_complete_version_custom_repo(self, module_org):
+        """Export a custom repository via complete version
 
         :id: 9c855866-b9b1-4e32-b3eb-7342fdaa7116
 
-        :expectedresults: Repository was successfully exported, rpm files are
+        :expectedresults: Repository was successfully exported, exported files are
+            present on satellite machine
+
+        :CaseLevel: System
+        """
+        # setup custom repo
+        cv_name = gen_string('alpha')
+        export_dir = '/var/lib/pulp/exports/'
+        product = make_product({'organization-id': module_org.id})
+        repo = make_repository(
+            {
+                'download-policy': 'immediate',
+                'organization-id': module_org.id,
+                'product-id': product['id'],
+            }
+        )
+        Repository.synchronize({'id': repo['id']})
+        # create cv and publish
+        cv = make_content_view({'name': cv_name, 'organization-id': module_org.id})
+        ContentView.add_repository(
+            {
+                'id': cv['id'],
+                'organization-id': module_org.id,
+                'repository-id': repo['id'],
+            }
+        )
+        ContentView.publish({'id': cv['id']})
+        cv = ContentView.info({'id': cv['id']})
+        assert len(cv['versions']) == 1
+        cvv = cv['versions'][0]
+        # Verify export directory is empty
+        result = ssh.command(f'find {export_dir} -name "*.json"')
```

Comment:
Please use `default_sat` fixture provided instance to call `default_sat.execute` instead of continuing to rely on the ssh module.  Given the scope of this change in this PR, it should be a very fast and straightforward change.